pkg/controller/beat/common/pod.go

```diff
@@ -61,7 +61,7 @@ func initContainerParameters(typ string) keystore.InitContainerParameters {
 		KeystoreCreateCommand:         fmt.Sprintf("%s keystore create --force", typ),
 		KeystoreAddCommand:            fmt.Sprintf(`cat "$filename" | %s keystore add "$key" --stdin --force`, typ),
 		SecureSettingsVolumeMountPath: keystore.SecureSettingsVolumeMountPath,
-		DataVolumePath:                fmt.Sprintf(DataPathTemplate, typ),
+		KeystoreVolumePath:            fmt.Sprintf(DataPathTemplate, typ),
```

Comment:
Also not your change, but `typ` is a typo of `type`, correct?

Reply:
`type` is a reserved word though, that's probably why it's `typ`.

Reply:
`typ` is acceptable even I'm agree it's disturbing at the first glance. Alternative ideas: `kind`, `beatType`, `specType`, `beatSpecType`.